clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1983,6 +1979,8 @@ class SyclKernelDeclCreator : public SyclKernelFieldHandler {
     if (isReadOnlyAccessor(AccessModeArg))
       Params.back()->addAttr(
           SYCLAccessorReadonlyAttr::CreateImplicit(SemaRef.getASTContext()));
+    Params.back()->addAttr(
+        SYCLAccessorPtrAttr::CreateImplicit(SemaRef.getASTContext()));
```

Comment:
Indention is off here:
```suggestion
   if (isReadOnlyAccessor(AccessModeArg))
      Params.back()->addAttr(
          SYCLAccessorReadonlyAttr::CreateImplicit(SemaRef.getASTContext()));
     Params.back()->addAttr(
         SYCLAccessorPtrAttr::CreateImplicit(SemaRef.getASTContext()));
```

Reply:
I believe @srividya-sundaram is correct here. The new code is not part of the `if`